### PR TITLE
A/B and MV testing

### PR DESCRIPTION
**Pull request by jonathanglassman**
Add the A/B and MV testing best practice from https://docs.google.com/document/d/1MAFk0JYh9wWs-bfkmhHv8V9RPpZL3kU7uS_6GLJoed8/edit# to the data community tech docs

### CODE REVIEW

File: source/analysis/abmv/index.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---
+title: A/B and multivariate testing
+weight: 47
+last_reviewed_on: 2022-05-12
+review_in: 6 months
+---
+
+# A/B and multivariate testing
+
+You can use A/B or multivariate (MV) testing to test hypotheses and optimise webpage.
```
Comment (TMSGDS) [inline source/analysis/abmv/index.html.md.erb:10]: optimise webpages

File: source/analysis/abmv/index.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+In A/B testing, you create 2 or more complete different versions of a webpage, and split web traffic between those versions. You use A/B testing A/B testing to test complete different website ideas.
```
Comment (TMSGDS) [inline source/analysis/abmv/index.html.md.erb:12]: Duplicate "A/B testing" - remove one

File: source/analysis/abmv/index.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+In MV testing, you identify the important areas of a webpage, and then create variations of the webpage to test against each other. You use MV testing for optimise and refine an existing webpage without having to do a significant redesign.
```
Comment (TMSGDS) [inline source/analysis/abmv/index.html.md.erb:14]: Hmm I wouldn't say "you identify the important areas of a webpage" as that might not be the case, you might be testing an insignificant area. What about something like "you identify a few key sections of a page and then create variations for those sections to test against each other"

File: source/analysis/abmv/index.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+In both methods, you present users with a variant at random, and you use statistical analysis to determine which variant performs better for a defined conversion goal.
+
+## How to do A/B and MV testing
+
+There are 7 steps to doing A/B and MV testing:
```
Comment (TMSGDS) [inline source/analysis/abmv/index.html.md.erb:20]: You've only put in 6 steps. So you can add in "test more" as the final step, or change this to 6

File: source/analysis/abmv/index.html.md.erb [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+Form the hypothesis in the format “If…, then…, because…”, where you would fill the blanks with “If [thing that you’re changing], then [result of change], because [rationale].”
+
+For example, “If I change how I signpost page A on page X, then I will see more users clicking through from page X to page A because they’ve been able to find the correct content they need which, according to an analysis of user journeys in GA and user feedback, users have been struggling with”.
+
+The hypothesis shows that you’ve really considered what you’re testing and why, and should be reliably documented somewhere. Some A/B testing tools allow you to store hypotheses in their tools.
+
+## Design
+
+Now you have created a hypothesis, you design the test to test it.
```
Comment (TMSGDS) [inline source/analysis/abmv/index.html.md.erb:51]: I'd change this to "Now you have created a hypothesis, you design a test to prove or disprove it."

File: source/analysis/abmv/index.html.md.erb [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+For the previous example, you could sketch adding a banner to page X or changing the font colour of the link in question.
+
+At this stage, if part of a product sprint, this is where you'd play “crazy eights”.
```
Comment (TMSGDS) [inline source/analysis/abmv/index.html.md.erb:55]: Should we link out to crazy 8s here? Not sure how wildly this is used in GDS - https://blog.prototypr.io/how-to-run-a-crazy-eights-workshop-60d0a67b29a

File: source/analysis/abmv/index.html.md.erb (hunk 3 above)
Comment (jonathanglassman) [reply source/analysis/abmv/index.html.md.erb:14]: Agree. Will substitute "key" for "important" in line with the style guide

Comment (jonathanglassman) [reply source/analysis/abmv/index.html.md.erb:14]: So will have a think and draft something appropriate

File: source/analysis/abmv/index.html.md.erb (hunk 6 above)
Comment (jonathanglassman) [reply source/analysis/abmv/index.html.md.erb:55]: probably a good idea

**Pull request merged**